### PR TITLE
Add Cjk bigram token filter page

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Add Cjk bigram token filter page

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/7874

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by kolchfa-aws** [comment]
@udabhas Could you review this PR for technical accuracy?

**Comment by vagimeli** [comment]
@varun-lodaya The documentation is awaiting tech review and approval, which is delaying progress. Could you please suggest alternative reviewers who can assist with this task in a timely manner? We're eager to move this forward. Thank you.

**Review by udabhas — APPROVED** [review]
Doc looks good to me

**Comment by vagimeli** [comment]
> Doc looks good to me

Thank you for the review. Moving this forward.

**Review by kolchfa-aws — COMMENTED** [review]
Thank you, @AntonEliatra! Some comments/suggestions.

**Comment by AntonEliatra** [comment]
@kolchfa-aws thats been updated now

**Review by kolchfa-aws — APPROVED** [review]
LGTM. Thank you, @AntonEliatra!

File: _analyzers/token-filters/cjk-bigram.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+---
+layout: default
+title: CJK bigram
+parent: Token filters
+nav_order: 30
+---
+
+# CJK bigram token filter
+
+The `cjk_bigram` token filter is designed specifically for processing East Asian languages, such as Chinese, Japanese, and Korean (CJK), which typically don't use spaces to separate words. A bigram is a sequence of two adjacent elements from a string of tokens, which can be characters or words. For CJK languages, bigrams help approximate word boundaries and capture significant character pairs that can convey meaning.
```
Comment (natebower) [inline _analyzers/token-filters/cjk-bigram.md:10]:
```suggestion
The `cjk_bigram` token filter is designed specifically for processing East Asian languages, such as Chinese, Japanese, and Korean (CJK), which typically don't use spaces to separate words. A bigram is a sequence of two adjacent elements in a string of tokens, which can be characters or words. For CJK languages, bigrams help approximate word boundaries and capture significant character pairs that can convey meaning.
```

File: _analyzers/token-filters/cjk-bigram.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+## Parameters
+
+The `cjk_bigram` token filter can be configured with two parameters: `ignore_scripts`and `output_unigrams`.
+
+### `ignore_scripts`
+
+The `cjk-bigram` token filter ignores all non-CJK scripts (writing systems like Latin or Cyrillic) and tokenizes only CJK text into bigrams. Use this option to specify CJK scripts to be ignored. This option takes the following valid values:
+
+- `han`: The `han` script is used to handle han characters. [Han characters](https://simple.wikipedia.org/wiki/Chinese_characters) are logograms used in the written languages of China, Japan, and Korea. The filter can help in text processing tasks like tokenizing, normalizing, or stemming text written in Chinese, Japanese Kanji, or Korean Hanja.
```
Comment (natebower) [inline _analyzers/token-filters/cjk-bigram.md:21]:
```suggestion
- `han`: The `han` script processes Han characters. [Han characters](https://simple.wikipedia.org/wiki/Chinese_characters) are logograms used in the written languages of China, Japan, and Korea. The filter can help with text processing tasks like tokenizing, normalizing, or stemming text written in Chinese, Japanese Kanji, or Korean Hanja.
```

File: _analyzers/token-filters/cjk-bigram.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+- `hangul`: The `hangul` script is specific to the Hangul script, which is the alphabet used to write the Korean language. This filter is useful for processing Korean text by handling Hangul syllables, which are unique to Korean and do not exist in other East Asian scripts.
```
Comment (natebower) [inline _analyzers/token-filters/cjk-bigram.md:23]:
```suggestion
- `hangul`: The `hangul` script processes Hangul characters, which are unique to the Korean language and do not exist in other East Asian scripts.
```

File: _analyzers/token-filters/cjk-bigram.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+- `hiragana`: The `hiragana` script is used for processing Hiragana, one of the two syllabaries used in the Japanese writing system.
```
Comment (natebower) [inline _analyzers/token-filters/cjk-bigram.md:25]:
```suggestion
- `hiragana`: The `hiragana` script processes Hiragana, one of the two syllabaries used in the Japanese writing system.
```

File: _analyzers/token-filters/cjk-bigram.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    Hiragana is typically used for native Japanese words, grammatical elements, and certain forms of punctuation.
+
+- `katakana`: The `katakana` script is for Katakana, the other syllabary used in Japanese.
```
Comment (natebower) [inline _analyzers/token-filters/cjk-bigram.md:28]:
```suggestion
- `katakana`: The `katakana` script processes Katakana, the other Japanese syllabary.
```

File: _analyzers/token-filters/cjk-bigram.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+```json
+POST /cjk_bigram_example/_analyze
+{
+  "analyzer": "cjk_bigrams_no_katakana",
+  "text": "東京タワーに行く"
+}
+```
+{% include copy-curl.html %}
+
+Sample Text: "東京タワーに行く"
```
Comment (natebower) [inline _analyzers/token-filters/cjk-bigram.md:79]:
```suggestion
Sample text: "東京タワーに行く"
```

**Review by natebower — COMMENTED** [review]
@AntonEliatra @kolchfa-aws Please see my changes and let me know if you have any questions. Thanks!

**Comment by kolchfa-aws** [comment]
@AntonEliatra Please resolve merge conflicts so we can merge. Thanks!

**Comment by AntonEliatra** [comment]
@kolchfa-aws thats been resolved now, thanks

**Pull request merged**